generators/client/templates/webapp/app/components/Home.vue.ejs

```diff
@@ -4,38 +4,38 @@
             <span class="hipster img-fluid rounded"></span>
         </div>
         <div class="col-md-9">
-            <h1 class="display-4" v-text="$t('home.title')">Welcome, Java Hipster!</h1>
-            <p class="lead" v-text="$t('home.subtitle')">This is your homepage</p>
+            <h1 class="display-4" <%_ if (enableTranslation) { _%>v-text="$t('home.title')"<%_ } _%>>Welcome, Java Hipster!</h1>
+            <p class="lead" <%_ if (enableTranslation) { _%>v-text="$t('home.subtitle')"<%_ } _%>>This is your homepage</p>
```

Comment:
This will add too many ejs if blocks, can we do it with regex like in angular and react?

Reply:
Yes, there is a lot of ejs if block, I didn't know any better way to achieve this. Could you explain me what do you mean by "do it with regex like in angular and react?" ?
I've read react/angular templates and I didn't find any evidences. 

Would it be better to remove this snippet after generation like we do with needles ?

Reply:
In angular/react we process the templates to remove translation content here https://github.com/jhipster/generator-jhipster/blob/master/generators/utils.js look at the `replaceTranslation` for example, this way we can avoid having ejs blocks as much as possible since they are harder to maintain

Reply:
some replacements are done here as well https://github.com/jhipster/generator-jhipster/blob/master/generators/generator-base.js#L1842

Reply:
Ok, I got it. It should be nicer indeed, I'll handle this :) Thanks

Reply:
It's cleaner now :)

Reply:
@deepu, is that ok for you? Could I merge it?